docs/setup/settings.asciidoc

```diff
@@ -82,6 +82,7 @@ to 0 to disable.
 `elasticsearch.startupTimeout:`:: *Default: 5000* Time in milliseconds to wait for Elasticsearch at Kibana startup before
 retrying.
 `pid.file:`:: Specifies the path where Kibana creates the process ID file.
+`path.data`:: *Default: `./data`* The path where Kibana stores persistent data not saved in Elasticsearch
```

Comment:
All the other lines here have periods at the end. This one doesn't!

Reply:
I pushed a fix at https://github.com/elastic/kibana/pull/13514/commits/1d4e419644bdd3dcaa6691d8fbf6126f12a92eb3.

Reply:
Different PR, https://github.com/elastic/kibana/pull/13514.